lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
@dbarashev am I understanding that right, that the goal is to be able to somehow (doesn't matter how) load KaTeX object on demand without needing to execute all the code in katex.js every time?